classes/DepositObjectDAO.inc.php

```diff
@@ -113,26 +106,23 @@ public function markHavingUpdatedContent($journalId, $objectType) {
 					GROUP BY pdo.deposit_object_id',
 					['issueId', STATUS_PUBLISHED, (int) $journalId]
 				);
-				while (!$result->EOF) {
-					$row = $result->GetRowAssoc(false);
-					$depositObject = $this->getById($journalId, $row['deposit_object_id']);
+				foreach ($result as $row) {
+					$depositObject = $this->getById($journalId, $row->deposit_object_id);
 					$deposit = $depositDao->getById($depositObject->getDepositId());
-					if($deposit->getSentStatus() || !$deposit->getTransferredStatus()) {
+					//if ($deposit->getSentStatus() || !$deposit->getTransferredStatus()) {
```

Comment:
Commented code; is this intentional?

Reply:
Yes. It's to compare the before state and after state when reviewed in fact.

I think that we should repackage and send the deposit to the staging server if it has new content, no matter what it's current state. But I needed your opinion on that.

Reply:
I can return the code to the previous state for that (as it is not related to 3.3 port) and consider that along with the other pending changes.

Reply:
Ah, OK, thanks!

Reply:
@asmecher OK I removed the change

Reply:
I will merge it and create a release based on the master branch.